src/Quantum.jl

```diff
@@ -110,7 +110,7 @@ Returns the sites of a [`Quantum`](@ref) Tensor Network.
 """
 function sites(tn::Quantum; kwargs...)
     if isempty(kwargs)
-        collect(keys(tn.sites))
+        sort(collect(keys(tn.sites)))
```

Comment:
It's better if you use `sort!(collect(...))` to not allocate twice or `sort(keys(...))` to allocate just once when sorting.

```suggestion
        sort(keys(tn.sites))
```

Reply:
maybe it's better to just use sort!, if I use your suggestion 
```
> sort(keys(Quantum(psi).sites))
ERROR: MethodError: no method matching sort(::Base.KeySet{Site, Dict{Site, Symbol}})
The function `sort` exists, but no method is defined for this combination of argument types.
```

Reply:
ok, then use `sort!(collect(...))`